MVVMKit/View Controllers/MVVMDiffableCollectionViewController.swift

```diff
@@ -23,6 +23,8 @@
  */
 
 import UIKit
+
+#if canImport(Combine)
```

Comment:
I would add the whole file content inside `#if canImport(Combine)` including `import UIKit`.
This would allow the whole file to be skipped if Combine isn't available.